pkg/sdk/security_integrations_gen.go

```diff
@@ -487,14 +487,17 @@ type ScimIntegrationSet struct {
 	Enabled       *bool                    `ddl:"parameter" sql:"ENABLED"`
 	NetworkPolicy *AccountObjectIdentifier `ddl:"identifier,equals" sql:"NETWORK_POLICY"`
 	SyncPassword  *bool                    `ddl:"parameter" sql:"SYNC_PASSWORD"`
-	Comment       *string                  `ddl:"parameter,single_quotes" sql:"COMMENT"`
+	Comment       *StringAllowEmpty        `ddl:"parameter" sql:"COMMENT"`
+}
+
+type StringAllowEmpty struct {
```

Comment:
it seems like something that should be part of commons; it would be nice to add a comment documenting its purpose

Reply:
Moved to `common_types.go` and added a comment.